drivers/include/ili9341.h

```diff
@@ -93,20 +93,31 @@ extern "C" {
 #endif
 /** @} */
 
+/**
+ * @brief   Display rotation mode
+ */
+typedef enum {
+    ILI9341_ROTATION_VERT,      /**< Vertical mode */
+    ILI9341_ROTATION_VERT_FLIP, /**< Vertical flipped mode */
+    ILI9341_ROTATION_HORZ,      /**< Horizontal mode */
+    ILI9341_ROTATION_HORZ_FLIP, /**< Horizontal flipped mode */
+} ili9341_rotation_t;
```

Comment:
Why not

```suggestion
#define ILI9341_ROTATION_VERT       ILI9341_MADCTL_VERT
#define ILI9341_ROTATION_VERT_FLIP  ILI9341_MADCTL_VERT_FLIP
#define ILI9341_ROTATION_HORZ       ILI9341_MADCTL_HORZ
#define ILI9341_ROTATION_HORZ_FLIP  ILI9341_MADCTL_HORZ_FLIP
```

and get rid of `_ili9341_calc_rotation()`